modules/cudf/src/series/string.ts

```diff
@@ -96,6 +128,15 @@ export class StringSeries extends Series<Utf8String> {
    * https://docs.rapids.ai/api/libcudf/stable/md_regex.html
    *
    * A RegExp may also be passed, however all flags are ignored (only `pattern.source` is used)
+   *
+   * @example
+   * ```typescript
+   * import {Series} from '@rapidsai/cudf';
+   * const a = Series.new(["foo_bar", "bar_foo"]);
+   *
+   * a.matchesRe(/foo/ig) // [true, false]
+   * a.matchesRe('bar') // [false, true]
+   * ```
```

Comment:
I think the regex examples would benefit from some more realistic (but simple) words/sentences. Regex is abstract enough as it is without operating on dev-jargon

Reply:
added simpler, elaborate examples